zshrc

```diff
@@ -45,12 +45,13 @@ unsetopt nomatch
 
 # vi mode
 bindkey -v
-bindkey "^F" vi-cmd-mode
-bindkey jj vi-cmd-mode
```

Comment:
`Esc` still works. We could probably remove `jj`, and leave it up to local overrides (some people use `jk`-smash).